### PR TITLE
updates to helm charts

### PR DESCRIPTION
**Pull request by r0fls**
My bad, a few more typos/issues here:

- typos in namespace (missing colon)
- update redis service name
- missing action cassandra pw secrets

### CODE REVIEW

File: helm-lightweight/values.yaml [hunk 1]
```diff
@@ -34,6 +34,10 @@ action:
     replicas: 1
     image: local/k8guard-discover
     imagePullPolicy: IfNotPresent
+  secrets:
+    # passwords need to be base64, echo -n "cassandra" | base6
```
Comment (medyagh) [inline helm-lightweight/values.yaml]: you missed 4 at the end of the comment, it needs to be 
`echo -n "cassandra" | base64`

Comment (medyagh) [reply helm-lightweight/values.yaml]: is there way to seprate the secret files in helm ? so we can git ignore them and provide a .EXAMPLE secret file for them to rename? so the developers dont commit secret by accident ?

Comment (medyagh) [reply helm-lightweight/values.yaml]: like this file k8guard-start-from-here/minikube/action/k8guard-action-secrets.yaml.EXAMPLE which means we add  k8guard-start-from-here/minikube/action/k8guard-action-secrets.yaml to gitignore by default so no room for mistake ?

**Review by medyagh — CHANGES_REQUESTED** [review]

File: helm-lightweight/values.yaml (hunk 1 above)
Comment (r0fls) [reply helm-lightweight/values.yaml]: yes, helm will use multiple files if you do `helm upgrade -i deployment-name chart-name -f values.yaml -f secret-values.yaml`. So, maybe we should move the secrets to another file with the .EXAMPLE extension.

**Pull request closed without merging**